docs/user/install.md

```diff
@@ -80,6 +83,9 @@ configured. The Turbinia configuration file will be deployed on these instances
 as `etc/turbinia/turbinia.conf`. If you later want to increase the number of
 workers, you can edit the `turbinia/variables.tf` file mentioned above and
 re-run `terraform apply`
+To use Turbinia you need to use the virtual environment that was setup by
```

Comment:
I would probably say "you can use" instead of "need to" just to be clear.  Also, very minor nit, but the `deploy.sh` script is what sets up the virtualenv, and not actually terraform.

Reply:
done